src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts

```diff
@@ -30,6 +30,7 @@ export function mapAggregateToData(
         organisationId: personenKontext.organisationId,
         rolleId: rel(RolleEntity, personenKontext.rolleId),
         rolle: Rolle.LERNENDER, // Placeholder, until rolle is removed from entity
+        befristung: personenKontext.befristung ?? undefined,
```

Comment:
`?? undefined` is only needed because you used `Persisted<>` in the aggregate. You can remove this